build.gradle

```diff
@@ -111,13 +111,13 @@ subprojects {
   targetCompatibility = '1.8'
 
   dependencies {
-    implementation 'org.slf4j:slf4j-api'
+    implementation 'org.slf4j:slf4j-api:1.7.30'
```

Comment:
I believe we should specify the version in `version.props` or does that not work?

Reply:
+1 for versions.props, especially as this is a very importance dependency.

Reply:
done!